tests/api2/test_disk_get_dev_size.py

```diff
@@ -5,14 +5,13 @@
 
 from middlewared.test.integration.utils import call, ssh
 
-DISKS = None
+DISKS = list(call('device.get_disks').keys())
```

Comment:
The original PR ([8620](https://github.com/truenas/middleware/pull/8620)) had `DISKS = None` -- what's the difference for Angelfish ?

Reply:
Another PR went into master to fix that particular problem. It's an idiosyncrasy of pytest.

Reply:
Copy that... just wanted to make sure this was intentional 😁